src/test/java/edu/tamu/cap/controller/UserControllerTest.java

```diff
@@ -20,17 +22,15 @@
 import org.springframework.boot.test.mock.mockito.MockBean;
 import org.springframework.http.MediaType;
 import org.springframework.security.test.context.support.WithMockUser;
-import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
 import org.springframework.test.web.servlet.MockMvc;
 
-import com.fasterxml.jackson.databind.ObjectMapper;
-
 import edu.tamu.cap.model.User;
 import edu.tamu.cap.model.repo.UserRepo;
 import edu.tamu.cap.utility.MockUserUtility;
 
 @SpringBootTest(webEnvironment = WebEnvironment.RANDOM_PORT)
-@RunWith(SpringRunner.class)
+@ExtendWith(SpringExtension.class)
```

Comment:
`@SpringBootTest` already includes this, please remove `ExtendWith(SpringExtension.class)`.